var/spack/repos/builtin/packages/python/package.py

```diff
@@ -670,6 +670,10 @@ def libs(self):
         # to ask Python where its LIBDIR is.
         libdir = self.get_config_var('LIBDIR')
 
+        # In Ubuntu 16.04.6 and python 2.7.12 from the system, lib could be
+        # in LBPL
```

Comment:
Can you provide some reference url in the comment?

Reply:
Done